src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,280 @@
+        } = person;
+        const modifieedPerson = { ...person };
+
+        if (!motherName) {
+          modifieedPerson.motherName = '-';
+        }
+
+        if (!fatherName) {
+          modifieedPerson.fatherName = '-';
+        }
+
+        modifieedPerson.mother = result.find(({ name }) => name === motherName);
+        modifieedPerson.father = result.find(({ name }) => name === fatherName);
+
+        return modifieedPerson;
+      });
+
+      setPeople(modifiedPeople);
+    } catch {
+      setIsError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    getPeopleFromServer();
+  }, []);
+
+  const filteredPeople = useMemo(
+    () => people.filter(({
+      name,
+      fatherName,
+      motherName,
+      sex,
+      born,
+    }) => {
+      const nameQuery = SPquery !== null
+        ? SPquery.trim().toLowerCase()
+        : null;
+
+      const nameFilter = nameQuery === null
+        || name.toLowerCase().includes(nameQuery)
+        || fatherName?.toLowerCase().includes(nameQuery)
+        || motherName?.toLowerCase().includes(nameQuery);
+
+      const sexFilter = SPsex === null
+        || sex === SPsex;
+
+      const bornFilter = !SPcenturies.length
+        || SPcenturies.includes(Math.floor(born / 100 + 1).toString());
+
+      return nameFilter && sexFilter && bornFilter;
+    }),
+    [people, SPquery, SPsex, SPcenturies],
+  );
+
+  if (sort !== null) {
+    filteredPeople.sort((personOne, personTwo) => {
+      const valueOne = personOne[sort];
+      const valueTwo = personTwo[sort];
+      let diffrence = 0;
+
+      switch (typeof valueOne) {
+        case 'number':
+          if (typeof valueTwo === 'number') {
+            diffrence = valueOne - valueTwo;
+          }
+
+          break;
+        case 'string':
+          if (typeof valueTwo === 'string') {
+            diffrence = valueOne.localeCompare(valueTwo);
+          }
+
+          break;
+        default: return 0;
+      }
+
+      return order ? -diffrence : diffrence;
+    });
+  }
 
-export const PeoplePage = () => {
   return (
-    <>
+    <div className="container">
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+          {!isLoading && (
+            <div className="column is-7-tablet is-narrow-desktop">
+              <PeopleFilters />
+            </div>
+          )}
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
+              {isLoading && <Loader />}
+
+              {isError && (
+                <p data-cy="peopleLoadingError" className="has-text-danger">
+                  Something went wrong
+                </p>
+              )}
+
+              {!isLoading && !isError && !people.length && (
```

Comment:
Use Boolean() when you can have zero
```suggestion
              {Boolean(!isLoading && !isError && !people.length) && (
```
because React renders 0 as a value.
![image](https://user-images.githubusercontent.com/45945732/233782133-896d9434-7164-44b0-a491-7f31e333803e.png)